### PR TITLE
Set key prop values

### PR DESCRIPTION
**Pull request by alstojanovic**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: src/domain/shared/components/ActivityLog/ActivityComponent.tsx [hunk 1]
```diff
@@ -63,9 +63,10 @@ export const ActivityComponent: FC<ActivityLogComponentProps> = ({ activities =
   const display = useMemo(() => {
     return (
       <>
-        {activities.map(activity => (
-          <ActivityViewChooser activity={activity} journeyLocation={journeyLocation} />
-        ))}
+        {activities.map(activity => {
+          const key = new Date(activity.createdDate).getTime();
```
Comment (hero101) [inline src/domain/shared/components/ActivityLog/ActivityComponent.tsx:67]: the point of the key is to not change over time so React does not re-render the element

**Review by hero101 — CHANGES_REQUESTED** [review]

File: src/domain/shared/components/ActivityLog/ActivityComponent.tsx (hunk 1 above)
Comment (techsmyth) [reply src/domain/shared/components/ActivityLog/ActivityComponent.tsx]: surely the key should just be the activity.id?

**Review by hero101 — APPROVED** [review]

File: src/domain/shared/components/ActivityLog/ActivityComponent.tsx (hunk 1 above)
Comment (alstojanovic) [reply src/domain/shared/components/ActivityLog/ActivityComponent.tsx]: we sorted it out. The problem was that id wasn't fetched with the query.

Comment (techsmyth) [reply src/domain/shared/components/ActivityLog/ActivityComponent.tsx]: Super :)

**Pull request merged**